docs/reference/search/search-your-data/search-shard-routing.asciidoc

```diff
@@ -88,6 +88,14 @@ GET /my-index-000001/_search?preference=my-custom-shard-string
 ----
 // TEST[setup:my_index]
 
+WARNING: If a node has <<shard-allocation-awareness,shard allocation awareness>>
+attributes, then it will prefer shards with the same awareness attribute values.
+So even if a search specifies the same custom `preference` values, different
+coordinating nodes may route it differently because of their awareness
+attributes. You can disable routing based on awareness attributes entirely by
+enabling the `es.search.ignore_awareness_attributes` system property on every
+node in the cluster.
```

Comment:
A couple of minor nits:

* I found the second sentence a little awkward. I tried to reword to fix.
* I added a link to our JVM options in the last sentence to point users to the best way to configure system properties.

```suggestion
Because of these attributes, different coordinating nodes may route same search
differently, even if the searches use the same custom `preference` values. To
disable routing based on awareness attributes, set the
`es.search.ignore_awareness_attributes` system property to `true` in the
<<set-jvm-options,JVM options>> of every node in the cluster.
```

Reply:
I like the new wording, thanks!